mirror-service/src/main/java/org/hyades/vulnmirror/datasource/osv/OsvToCyclonedxParser.java

```diff
@@ -99,8 +99,12 @@ private static List<VulnerabilityAffects> parseAffectedRanges(JSONArray osvAffec
             JSONObject osvAffectedObj = osvAffectedArray.getJSONObject(i);
             String purl = parsePackageUrl(osvAffectedObj);
             try {
-                packageUrl = new PackageURL(purl);
-                ecoSystem = packageUrl.getType();
+                if (purl != null) {
+                    packageUrl = new PackageURL(purl);
+                    ecoSystem = packageUrl.getType();
+                } else {
+                    LOGGER.debug("Package url was null");
```

Comment:
I think this can safely be removed. Without context (e.g. in which OSV advisory it occurred), there's no value in being notified about PURLs being null.

Reply:
removed